gemsets/ruby/3.0.6/global.gems

```diff
@@ -0,0 +1 @@
+openssl
```

Comment:
Can you add CHANGELOG entry please?

Reply:
Btw, i don't see changelog entry for previous versions